packages/core/src/analytics.ts

```diff
@@ -558,7 +558,7 @@ export class SegmentClient {
     this.logger.info('TRACK event saved', event);
   }
 
-  identify(userId: string, userTraits?: UserTraits) {
+  identify(userId: string | undefined, userTraits?: UserTraits) {
```

Comment:
For consistency, lets use the optional as:
```suggestion
  identify(userId?: string, userTraits?: UserTraits) {
```